packages/ods/src/components/input/tests/rendering/ods-input.e2e.ts

```diff
@@ -58,15 +61,38 @@ describe('ods-input rendering', () => {
         <button type="reset">Reset</button>
         <button type="submit">Submit</button>
       </form>`);
+
       const resetButton = await page.find('button[type="reset"]');
       await resetButton.click();
 
       const submitButton = await page.find('button[type="submit"]');
       await submitButton.click();
       await page.waitForNetworkIdle();
+
       const url = new URL(page.url());
       expect(url.searchParams.get('natifInput')).toBe('');
       expect(url.searchParams.get('odsInput')).toBe('');
     });
   });
+
+  describe('slots', () => {
+    describe('list', () => {
```

Comment:
need a test with list attributes but no slot datalist (with spy on console.warn) and one with the list of option changed